server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -476,6 +481,7 @@ private List<UploadedIndexMetadata> writeIndexMetadataParallel(ClusterState clus
                 )
             );
             exceptionList.forEach(exception::addSuppressed);
+            logger.error("Exception in transferring index metadata to remote store", exception);
```

Comment:
this exception is already caught in `RemotePersistedState.setLastAcceptedState` so we might be doing duplicate logging

Reply:
Can you confirm if it will be logged in there or not, I couldn't logs in there. 

Also we can hit this error during prePublish as well Right?

Reply:
yeah, this would get logged as the pre publish would fail

Reply:
I checked the code, these exceptions are propagated to Master Service and it is getting logged over there. It will be logged as part of this log lines: 
https://github.com/opensearch-project/OpenSearch/blob/3e64a7b92192a37546b92b811e04c4155dafbdfd/server/src/main/java/org/opensearch/cluster/service/MasterService.java#L398-L405

I will remove these error log from both IndexMetadata and GlobalMetadata.